fixtures/flight/config/webpack.config.js

```diff
@@ -664,7 +664,14 @@ module.exports = function(webpackEnv) {
           formatter: isEnvProduction ? typescriptFormatter : undefined,
         }),
       // Fork Start
-      new ReactFlightWebpackPlugin({isServer: false}),
+      new ReactFlightWebpackPlugin({
+        isServer: false,
+        clientReferences: {
+          directory: './src/',
+          recursive: true,
+          include: /\.client\.js$/,
```

Comment:
Do you have a plan for `node_modules`? Am I supposed to exclude them here? Do they follow a different convention?

Reply:
You’re supposed to *include* them. That’s why this is such a bad idea and it’s better to compile server first and use the reachable ones as a list here.

Reply:
I think I'm still fuzzy on how you distribute Server+Client over npm. E.g. the default on npm today is Client. So if I import `react-something` from a Server one then I'd actually expect this to behave like a `.client` entry point because the package author doesn't know about the distinction.

Reply:
You will get a static error on the server because you’re trying to import an export from React that doesn’t exist like useState. At least with real ESM. (We might want to fast track ESM for that reason.)

Existing things that just work as hybrids continue to work. Others will error on the server and you have to wrap it in a .client file. Ideally the publisher would’ve already done that but if not, you have to add a local wrapper to fix the error.

Reply:
Note that I think it’s the *resolved* file that needs to be .client.js. So you can add an “exports” alias so the import is just plain.

That’s no different than a hybrid wrapper that references a client.

Reply:
makes sense